modules/checking-the-compliance-status-of-your-infrastructure.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-compliance.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-the-compliance-status-of-your-infrastructure_{context}"]
+= Checking the compliance status of your infrastructure
+
+By performing a compliance scan, you can check the compliance status of your entire infrastructure for all compliance standards. When you run a compliance scan, {rh-rhacs-first} creates a data snapshot of your environment. The data snapshot includes alerts, images, network policies, deployments, and related host-based data.
+
+Central collects the host-based data from Sensors running in your clusters. Central then collects further data from the compliance container running in each Collector pod. 
+
+The compliance container collects the following data about your environment:
+
+* Configurations for Docker Daemon, Docker image, and Docker container.
```

Comment:
```suggestion
* Configurations for the container daemon, container runtime and container image.
```

Reply:
Docker is the container-runtime (containerd more specifically) kubernetes uses. In OpenShift we do not use containerd, we use cri-o. Imho it is an implementation detail, we can refer to "container daemon" and not specify the runtime.